src/credit.ts

```diff
@@ -267,4 +268,21 @@ export default class Credit extends AinftBase {
     const trailingUrl = `deposit/transaction`;
     return this.sendRequest(HttpMethod.POST, trailingUrl, body);
   }
+
+    /**
+   * send deposit transaction to nft server
```

Comment:
send -> Send?
nft server -> AINFT Server?

period(.) at the end?

Reply:
I applied it. thanks!